react/Modal/index.jsx

```diff
@@ -313,6 +313,9 @@ class Modal extends Component {
     const style = Object.assign({}, height && { height }, width && { width })
     const maybeWrapInPortal = children =>
       into ? <Portal into={into}>{children}</Portal> : children
+    const hasModalHeader = children.find(
+      child => child.type.name === 'ModalHeader'
```

Comment:
I think this will crash on Drive saying `can't read name of undefined` since Preact doesn't expose `type` object for its children.

Reply:
Here is a quick fiddle to illustrate. https://jsfiddle.net/j40snyd1/ enjoy :D

Reply:
Do you know any alternative ?

Reply:
Also, when minified, name will change, type will not. 
```
 {this.props.children.find(({ type }) => type === ModalHeader)}
```

I think we should test if react / if preact and do similar thing.